abci/src/lib.rs

```diff
@@ -9,7 +9,7 @@
 //! have them interact. In practice, the client interaction will be performed
 //! by a full Tendermint node.
 //!
-//! ```rust
+//! ```ignore
```

Comment:
Was this doctest failing?

Reply:
Yup. `cargo test` on the project root fails because of this and a few other doctests.

Reply:
I've submitted #990 to fix these problems.

Reply:
It would be preferrable to not just ignore parts that break. The least would have been a tracking issue, a fix like @thanethomson provided is obviously preferrable.

Reply:
Nice! I will revert this change then.